Scripts/cmake/Platform/iOS/Toolchain_ios.cmake

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+set(CMAKE_SYSTEM_NAME iOS)
+set(CMAKE_SYSTEM_VERSION 1)
+set(CMAKE_OSX_ARCHITECTURES arm64)
+
+
+set(LY_IOS_CODE_SIGNING_IDENTITY "iPhone Developer" CACHE STRING "iPhone Developer")
```

Comment:
I'm not 100% certain, but all the PAL_ and LY_ attributes can be removed for 3rd party builds. This toolchain was copied from O3DE

Reply:
I'm not actually sure what you mean.  Isn't this used to make packages for ios?  It doesn't pull anything in from o3de.

Reply:
I think @spham-amzn means that the PAL_PLATFORM_NAME and PAL_PLATFORM_NAME_LOWERCASE is O3DE engine specific and is not needed for a general iOS build.

Reply:
Yes, same goes with the LY_* variables

Reply:
Do we have to remove the code signing then?
It has this
set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED YES)
set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY ${LY_IOS_CODE_SIGNING_IDENTITY})
set(CMAKE_XCODE_ATTRIBUTE_DEVELOPMENT_TEAM ${LY_IOS_DEVELOPMENT_TEAM})

Reply:
will remove and see if the current packages build